### PR TITLE
Feature/grid

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/features/shared/components/atoms/Grid/Row.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * optional props:
+ *    - isFullWidth: by default the max-width of content is set to 1430px, overwrite this with 100% by setting the isFullWidth = true
+ *    - marginBottom, marginTop: add vertical margins by use of the theme's spacing (this includes 'px')
+ * Implementation :
+ * <Row isFullWidth marginBottom={themeSpacing(10)} marginTop={themeSpacing(10)} >
+ */
+
+export type TypeProps = {
+  children: React.ReactNode
+  marginBottom?: any
```
Comment (gilleswittenberg) [inline src/app/features/shared/components/atoms/Grid/Row.tsx:18]: type is `number`

File: src/app/features/shared/components/atoms/Grid/Row.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  marginTop?: any
+  addColumn?: boolean
```
Comment (gilleswittenberg) [inline src/app/features/shared/components/atoms/Grid/Row.tsx:20]: Is this being used?

File: src/app/features/shared/components/atoms/Grid/Row.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  flex-wrap: wrap;
+  width: calc(100% + ${ GUTTER } px);
+  margin-left: -${ GUTTER / 2 }px;
+  margin-right: -${ GUTTER / 2 }px;
+  @media screen and ${ breakpoint("min-width", "laptop") } {
+    width: calc(100% + ${ GUTTER }px);
+    margin-left: -${ GUTTER / 2 }px;
+    margin-right: -${ GUTTER  / 2 }px;
+  }
+  margin-top: ${ props => props.marginTop || 0 };
```
Comment (gilleswittenberg) [inline src/app/features/shared/components/atoms/Grid/Row.tsx:39]: `${ props => props.marginTop ?? 0 }`

**Review by gilleswittenberg — APPROVED** [review]

File: src/app/features/shared/components/atoms/Grid/Row.tsx (hunk 1 above)
Comment (remcohoff) [reply src/app/features/shared/components/atoms/Grid/Row.tsx:18]: For consistency the margin is set with the use of `themeSpacing` (a util from ASC), which returns an `any`, eg "24px". When I set type to `number` or `string` I get a TS error.

File: src/app/features/shared/components/atoms/Grid/Row.tsx (hunk 2 above)
Comment (remcohoff) [reply src/app/features/shared/components/atoms/Grid/Row.tsx:20]: Nope, I removed it

File: src/app/features/shared/components/atoms/Grid/Row.tsx (hunk 3 above)
Comment (remcohoff) [reply src/app/features/shared/components/atoms/Grid/Row.tsx:39]: Done

File: src/app/features/shared/components/atoms/Grid/Row.tsx (hunk 1 above)
Comment (gilleswittenberg) [reply src/app/features/shared/components/atoms/Grid/Row.tsx:18]: Ok, than it's fine. Will check later.

**Pull request merged**